app/models/national_hearing_queue_entry.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class NationalHearingQueueEntry < ApplicationRecord
+  self.abstract_class = true
+end
```

Comment:
```suggestion
class NationalHearingQueueEntry < CaseflowRecord
end
```
I think that the CaseflowRecord class is used to gather descendants for our ERD diagrams on the github.io site.